src/services/users.service.ts

```diff
@@ -52,6 +52,17 @@ class UserService {
     const deleteUserData = await this.users.delete({ where: { id: userId }, select: { email: true, id: true } });
     return deleteUserData;
   }
+
+  public async manageGroups(userId: number, data: any) {
```

Comment:
We should avoid using any type for parameters when the type can be easily inferred or explicitly defined.